packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit/web_kit.dart

```diff
@@ -412,4 +416,79 @@ class WKWebView {
   Future<void> loadRequest(NSUrlRequest request) {
     throw UnimplementedError();
   }
+
+  /// Loads the contents of the specified HTML string and navigates to it.
+  Future<void> loadHtmlString(String string, String? baseUrl) {
```

Comment:
Here and below, we should make the second parameters named. As positional parameters the call sites are very hard to understand; much more than the corresponding ObjC or Swift versions. Making the second parameters named will make them more like the Swift APIs.